deye_docker_entrypoint.py

```diff
@@ -20,18 +20,20 @@
 
 from deye_cli import main as cli_main
 from deye_daemon import main as daemon_main
-from deye_config import DeyeConfig
+from deye_config import DeyeConfig, LOG_DEST_STDOUT, LOG_DEST_STDERR
 
-config = DeyeConfig.from_env()
-
-log_format = "%(asctime)s - %(name)s - %(levelname)s - %(message)s"
-
-logging.basicConfig(stream=sys.stdout, format=log_format, level=logging.getLevelName(config.log_level))
-
-log = logging.getLogger('main')
+def setupLogging(config):
+    log_format = "%(asctime)s - %(name)s - %(levelname)s - %(message)s"
 
+    log_stream = None
+    if config.log_stream == LOG_DEST_STDOUT : log_stream = sys.stdout
+    if config.log_stream == LOG_DEST_STDERR : log_stream = sys.stderr
```

Comment:
Please add basic validation here. My expectation is that the process will exit with status 1 if `config.log_stream` is set to unexpected value.

Reply:
Done.
PTAL